src/assets/index.js

```diff
@@ -17,3 +17,9 @@ export { default as whiteGoldPlate} from "./images/white-plate-gold-border.jpg"
 export { default as comboSet2} from "./images/combo-set-2.jpg";
 export { default as comboSet3} from "./images/combo-set-3.jpg";
```

Comment:
I like most the way you export and import the images and files. Great @ApurvaSawant11